spec/factories/user_profiles.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :user_profile do
```

Comment:
@sejalkhatri I agree that, at this point, keeping the factory independent is best. If it becomes an issue with further development of the related tests, we can reconsider it.